images/os/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM fedora:29 AS build
+
+COPY --from=registry.svc.ci.openshift.org/openshift/origin-v4.0:machine-os-content /srv/ /srv/
+RUN yum install -y ostree yum-utils selinux-policy-targeted && \
+    commit=$( find /srv -name *.commit | sed -Ee 's|.*objects/(.+)/(.+)\.commit|\1\2|' | head -1 ) && \
+    mkdir /tmp/working && cd /tmp/working && \
+    yumdownloader --destdir=/tmp/rpms origin-node origin-clients && \
+    for i in $(find /tmp/rpms/ -name origin-*); do rpm2cpio $i | cpio -div; done && \
+    ostree --repo=/srv/repo commit --parent=$commit --tree=ref=$commit --tree=dir=. --selinux-policy / \
+        -s "origin-ci-dev overlay RPMs" --branch=origin-ci-dev
```

Comment:
Ahh OK, so one final gotcha here is that we also need to do the `/etc` -> `/usr/etc` translation (yet another reason why we should teach this to rpm-ostree). So basically just:

```sh
    for i in $(find /tmp/rpms/ -name origin-*); do rpm2cpio $i | cpio -div; done && \
    mv etc usr/ && \
    ...
```